sql/core/src/test/resources/sql-tests/inputs/ansi/json-functions.sql

```diff
@@ -0,0 +1 @@
+--IMPORT json-functions.sql
```

Comment:
It's a bit overkill to rerun the entire test with ansi mode. What we really care about is the parsing of schema string.

How about we add a new test `parse-schema-string.sql`, which only tests `from_json`, `from_csv` with schema string?

Reply:
done